plugins/ComposeAuth/src/appleMain/kotlin/io/github/jan/supabase/compose/auth/composable/AppleAuth.kt

```diff
@@ -82,6 +85,7 @@ actual fun ComposeAuth.rememberSignInWithApple(
     return state
 }
 
+@BetaInteropApi
```

Comment:
@jan-tennert I don't know what this is, but I had to add it for using the `NSString.create` method.

Let me know if there is a better option.

Reply:
That should be fine